src/main/java/frc/robot/commands/LimeLightTurn.java

```diff
@@ -36,28 +36,25 @@ public LimeLightTurn() {
     // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
     }
 
+
+    private double targetTurn;
+
     // Called just before this Command runs the first time
     @Override
     protected void initialize() {
-        LimelightUtility.RefreshTrackingData();
-        //LimelightUtility.LogTrackingData();
         setTimeout(5);
```

Comment:
5 seconds seems high if limelight can't see target - is there way to check if target found and if not set timeout to lower?